.github/workflows/ci.yml

```diff
@@ -34,6 +34,9 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
 
+      - name: Format
+        run: diff -u <(echo -n) <(go fmt $(go list ./...))
```

Comment:
@segevda QQ: If format related files found, what happens? style failure?

Reply:
@jeevatkm not sure wdym by "style failure"?
The ci workflow will fail if there are unformatted files

Reply:
@segevda code formatting aka style. thanks for confirming.